src/styles/block/clock.scss

```diff
@@ -0,0 +1,49 @@
+.clock {
+  @include circle($clock-size);
+
+  display: flex;
+  margin: 50px auto;
+  position: relative;
+  background-color: gainsboro;
```

Comment:
Let's create variables for all colors too

Reply:
i'm fix please check my work))